library/src/main/java/com/novoda/downloadmanager/lib/BatchRepository.java

```diff
@@ -232,6 +235,18 @@ public void increment(int statusCode) {
         public void clear() {
             statusCounts.clear();
         }
+
+        @Override
+        public String toString() {
```

Comment:
No we don't but is a good practice to have it, I don't see any damage leaving it :)